Pipfile

```diff
@@ -4,14 +4,15 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-pytest = "==5.3.5"
-flake8 = "==3.7.9"
-pytest-mock = "==2.0.0"
+pytest = ">=5.3.5,<6.0.0"
+flake8 = ">=3.7.9,<4.0.0"
+pytest-mock = ">=2.0.0,<3.0.0"
 autopep8 = "*"
-pytest-cov = "==2.8.1"
-moto = "==1.3.8"
+pytest-cov = ">=2.8.1,<3.0.0"
+moto = ">=1.3.8,<2.0.0"
 
 [packages]
+alive-progress = ">=1.5.1"
```

Comment:
I see that you add a new dependency. Now we does not implemented something to update our setup.py to maintain updated the dependencies, can you add this lib on setup.py for that the installation of GGenerator deal with it and get it too?

Reply:
@iRitiLopes sure, thanks for bringing that up